src/app/screens/LoginScreen/layout.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { View, Image, ImageBackground, SafeAreaView, TextInput } from 'react-native';
+
+import background from '@assets/General/bc_inicio.png';
+import logo from '@assets/General/Group.png';
+import CustomButton from '@components/CustomButton';
+
+import styles from './styles';
+
+function LoginScreen({ onSubmit, onChangeEmail, onChangePassword, email, password }) {
+  return (
+    <SafeAreaView style={styles.container}>
+      <ImageBackground source={background} style={styles.imageBackground}>
+        <Image source={logo} style={styles.logo} />
+        <View styles={styles.loginContainer}>
+          <TextInput
+            style={styles.input}
+            onChangeText={(text) => onChangeEmail(text)}
```

Comment:
```suggestion
            onChangeText={onChangeEmail}
```

Same for the other one